.github/workflows/smoke-windows-strawberryperl.yml

```diff
@@ -26,22 +26,17 @@ jobs:
           perl -MIPC::Cmd -E "say IPC::Cmd::can_run(qq{gcc})"
           perl -MIPC::Cmd -E "say IPC::Cmd::can_run(qq{g++})"
           perl -MIPC::Cmd -E "say IPC::Cmd::can_run(qq{ld})"
-          path C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;%PATH%
+          echo "C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin" ^| Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
```

Comment:
This should be
```
echo "C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;$env:PATH" ^| Out-File -FilePath $env:GITHUB_PATH -Encoding utf8
```
The Strawberry Perl versions of Mingw need to be ahead of the Chocolately versions

Reply:
testing with your suggestion via 3aa63b90225c66fec1cddd23b3a108abd99f80a4